docs/source/testing/mocking.md

```diff
@@ -3,13 +3,13 @@ title: Mocking
 description: Mock your GraphQL data based on a schema.
 ---
 
-The strongly-typed nature of a GraphQL API lends itself extremely well to mocking. This is an important part of a GraphQL-First development process, because it enables frontend developers to build out UI components and features without having to wait for a backend implementation.
+Mocking allows Apollo Server to return simulated data for GraphQL operations based on your server's schema. The strongly-typed nature of a GraphQL API lends itself extremely well to mocking, which is an important part of a GraphQL-first development process.  
 
-Even when the UI is already built, it can let you test your UI without waiting on slow database requests, or build out a component harness using a tool like [Storybook](https://storybook.js.org/) without needing to start a real GraphQL server.
+Mocking enables frontend developers to build out and test UI components and features without having to wait for a full backend implementation. Mocking is also valuable when using a UI tool like [Storybook](https://storybook.js.org/), because you won't need to start a real GraphQL server.
 
-## Default mock example
+## Using default mocks
 
-This example demonstrates mocking a GraphQL schema with just one line of code, using `apollo-server`'s default mocking logic.
+You can turn on `apollo-server`'s default mocking logic by passing in a `mocks` option to the configuration of a new `ApolloServer` instance and setting that option to true. 
```

Comment:
`apollo-server`'s --> Apollo Server's (we only use the library name when talking about a _particular_ Apollo Server library since there are like 20 😄 )

true --> `true`

Suggest ending this with a colon instead of period because the immediately following codeblock demonstrates the concept

Reply:
"passing in a `mocks` option to the configuration of a new `ApolloServer` instance and setting that option to true" seems kinda verbose. "passing `mocks: true` to the `ApolloServer` constructor" perhaps?